CRM/Core/I18n/Schema.php

```diff
@@ -481,13 +483,15 @@ private static function createIndexQueries($locale, $table, $class = 'CRM_Core_I
    *   A DAO object to run DESCRIBE queries.
    * @param string $class
    *   schema structure class to use.
-   *
+   * @param bool $isUpgradeMode
+   *   Are we in upgrade mode therefore only build based off table not class
    * @return array
    *   array of CREATE INDEX queries
    */
-  private static function createViewQuery($locale, $table, &$dao, $class = 'CRM_Core_I18n_SchemaStructure') {
+  private static function createViewQuery($locale, $table, &$dao, $class = 'CRM_Core_I18n_SchemaStructure', $isUpgradeMode = FALSE) {
```

Comment:
@totten @mlutfy Folks do the changes below in this specific file make sense I was running into an issue where by it was looking for a field because it was in $columns[$table] but wasn't yet added to the table